pkg/ocm/users.go

```diff
@@ -78,3 +81,39 @@ func (c *Client) DeleteUser(clusterID string, group string, username string) err
 	}
 	return nil
 }
+
+func (c *Client) CreateRoleBinding(subscriptionID string, userName string, roleID string) (*amv1.RoleBinding, error) {
+	roleBinding, err := amv1.NewRoleBinding().Account(amv1.NewAccount().Username(userName)).
+		Role(amv1.NewRole().ID(roleID)).Build()
```

Comment:
@vkareh  this does not work yet.  I keep getting account_username is not set. I m going to work with SDB team/check the model to see if there is any else that is missed but wanted to get start on the PR review